### ISSUE TITLE
Education STEM Feature 508 Audit

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#724]
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/19125

**Comment by peterhill-oddball** [comment]
Unable to evaluate "get help" link from this page to : https://staging.va.gov/disability-benefits/apply/help/index.html as the page returns a 500 error.
![image.png](https://images.zenhubusercontent.com/5d2633e0a6db635dbcbaf8d5/a9c8546f-0230-4900-848e-506227880279)

**Comment by 1Copenut** [comment]
I'm opening 2 issue tickets for the Triage team that are more sitewide with respect to our forms that I noticed / re-noticed during the audit for this feature:

* https://github.com/department-of-veterans-affairs/va.gov-team/issues/767
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/317

**Comment by KevinMHoffmanUSDS** [comment]
Thanks, @1Copenut!

**Comment by 1Copenut** [comment]
Closing this as the audit has been done and all issues are remediated.
